_includes/layout/scripts

```diff
@@ -6,15 +6,10 @@
 <script src="/assets/js/util.js"></script>
 <script src="/assets/js/jquery.toc.js"></script>
 <script src="/assets/js/lightbox.min.js"></script>
-{% if page.usemathjax %}
-<script type="text/javascript" async
-  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=default">
-</script>
-{% endif %}
-{% if page.imjoy %}
-<script src="https://lib.imjoy.io/imjoy-loader.js"></script>
-<script src="/assets/js/imjoy-app.js"></script>
-{% endif %}
+<!-- Extensions -->
+{% for extension in page.extensions %}
+{% include extensions/{{ extension }} %}
```

Comment:
Ahh, nested Liquid includes! I didn't expect this to work. But glad it does.

Reply:
If someone added a non-existing key, would that cause error?

Reply:
Yep, if they did that, it would break the site build. But so does any Liquid syntax error. Maybe we can find a way to tell the Jekyll build to be tolerant of page build failures, but for now I'm not going to worry about it.